createdisk.sh

```diff
@@ -255,6 +255,10 @@ ${OC} --config $1/auth/kubeconfig create secret generic htpass-secret --from-lit
 ${OC} --config $1/auth/kubeconfig apply -f htpasswd_cr.yaml
 ${OC} --config $1/auth/kubeconfig create clusterrolebinding developer --clusterrole=sudoer --user=developer
 
+# Get cluster-kube-apiserver-operator image along with hash and tag it
+certImage=$(${OC} --config $1/auth/kubeconfig adm release info --image-for=cluster-kube-apiserver-operator)
```

Comment:
Is this going to use `quay.io/openshift-release-dev/ocp-release:4.2.2` by default? (with '4.2.2' being the openshift release we are building the bundle for)
The command I was using so far was
```
${OC} --config $1/auth/kubeconfig adm release info  quay.io/openshift-release-dev/ocp-release:4.2.2 --image-for=cluster-kube-apiserver-operator
```

Reply:
@cfergeau it is going to use whatever version we deployed on the cluster so for each release it should be it's own version since this is available from 4.2.0 so 4.2.2/4.2.4. ..etc. all have it's own `cluster-kube-apiserver-operator` image.